docs/sources/getting-started/get-started-grafana-prometheus.md

```diff
@@ -43,18 +54,47 @@ Prometheus node_exporter is a widely used tool that exposes system metrics. Inst
         # scheme defaults to 'http'.
 
           static_configs:
-          - targets: ['<hostname>:9100']
+          - targets: ['localhost:9100']
    ```
 
 1. Start the Prometheus service:
+
    ```
     ./prometheus --config.file=./prometheus.yml
    ```
 
-#### Check Prometheus metrics in Grafana Explore view
+1. Confirm that Prometheus is running by navigating to `http://localhost:9090`. You can see that the node_exporter metrics have been delivered to Prometheus. Next we will send them to Grafana.
+
+#### 4. Configure Prometheus for Grafana
+
+When running Prometheus locally, there are two ways to configure Prometheus for Grafana. You may use a hosted Grafana instance at [Grafana Cloud](https://grafana.com/) or run Grafana locally. This guide will describe configuring Prometheus in a hosted Grafana instance on Grafana Cloud.
+
+1. Sign up for [https://grafana.com/](https://grafana.com/auth/sign-up/create-user). Grafana gives you a Prometheus instance out of the box.
+
+SHOW PICTURE HERE
```

Comment:
@bohandley , images are stored in the website repo and then you can add a reference to that image here. For image guidelines, take a look at https://grafana.com/docs/writers-toolkit/writing-guide/image-guidelines/.